src/main/resources/edu/cornell/kfs/tax/batch/Default1042STaxOutputDefinition.xml

```diff
@@ -29,83 +29,87 @@
+        <field name="B50" length="2" type="VENDOR_FOREIGN_ADDRESS" value="vendorCountryCode" /><!-- B50 :: Tax Residence Country Code -->
+        <field name="B51" length="25" type="BLANK" /><!-- B51 :: Tax Residence Country Name (NOTE: Should be non-blank if B43 is blank or invalid) -->
+        <field name="B52" length="10" type="BLANK" /><!-- B52 :: Passport Expiration Date -->
+        <field name="B53" length="10" type="STATIC" value="KFS" /><!-- B53 :: Record Source -->
+        <field name="B54" length="20" type="BLANK" /><!-- B54 :: Campus -->
+        <field name="B55" length="9" type="DERIVED" value="ein" /><!-- B55 :: EIN -->
+        <field name="B56" length="1" type="BLANK" /><!-- B56 :: Use US Address -->
+        <field name="B57" length="1" type="BLANK" /><!-- B57 :: Spouse_dependant -->
+        <field name="B58" length="2" type="BLANK" /><!-- B58 :: State of employment -->
+        <field name="B59" length="40" type="BLANK" /><!-- B59 :: Occupation 2 -->
+        <field name="B60" length="1" type="BLANK" /><!-- B60 :: Personal exemption -->
+        <field name="B61" length="10" type="BLANK" /><!-- B61 :: User 1 (Text) -->
+        <field name="B62" length="10" type="BLANK" /><!-- B62 :: User 2 (Text) -->
+        <field name="B63" length="10" type="BLANK" /><!-- B63 :: User 3 (Currency) -->
+        <field name="B64" length="1" type="BLANK" /><!-- B64 :: User 4 (Check box) -->
+        <field name="B65" length="1" type="BLANK" /><!-- B65 :: User 5 (Check box) -->
+        <field name="B66" length="10" type="BLANK" /><!-- B66 :: User 6 (Date) -->
+        <field name="B67" length="10" type="BLANK" /><!-- B67 :: User 7 (Date) -->
+        <field name="B68" length="10" type="BLANK" /><!-- B68 :: User 8 (Currency) -->
+        <field name="B69" length="4" type="BLANK" /><!-- B69 :: Intermediary/FLW-THR ID -->
+        <field name="B70" length="40" type="BLANK" /><!-- B70 :: Intermediary/FLW-THR Name Line 1 -->
+        <field name="B71" length="4" type="BLANK" /><!-- B71 :: Payer ID -->
+        <field name="B72" length="40" type="BLANK" /><!-- B72 :: Payer Name -->
+        <field name="B73" length="40" type="BLANK" /><!-- B73 :: Maiden Name -->
+        <field name="B74" length="1" type="BLANK" /><!-- B74 :: LPR steps -->
+        <field name="B75" length="1" type="BLANK" /><!-- B75 :: Tax Home Connection -->
+        <field name="B76" length="1" type="BLANK" /><!-- B76 :: Claim Treaty benefits -->
+        <field name="B77" length="4" type="BLANK" /><!-- B77 :: Primary Withholding Agent's ID -->
+        <field name="B78" length="40" type="BLANK" /><!-- B78 :: Primary Withholding Agent's Name -->
         <!--
-            ==== NOTE: The current spec does not have any fields with names B75 through B96. ====
+            ==== NOTE: The current spec does not have any fields with names B79 through B96. ====
          -->
         <field name="B97" length="25" type="BLANK" /><!-- B97 :: Filler -->

```

Comment:
It may not be needed, but it doesn't seem to hurt anything, and if we leave as is, we could easily convert back to fixed width instead of tab delimited (although whether we will have that need is another question). Do you think there's a downside to including it and we should remove it?